RLTest/__main__.py

```diff
@@ -454,6 +462,9 @@ def __init__(self):
         Defaults.cluster_node_timeout = self.args.cluster_node_timeout
         Defaults.enable_debug_command = self.args.enable_debug_command
         Defaults.enable_protected_configs = self.args.enable_protected_configs
+        Defaults.enable_module_command = self.args.enable_module_command
+        Defaults.allow_unsafe = self.args.allow_unsafe
```

Comment:
I believe this can just enable the other three values? Will it make the implementation easier?